### ISSUE TITLE
use my own dataset and get loss=0.0

### ISSUE THREAD

**Issue by booyoungxu** [sthalles/deeplab_v3#21]
Hi,
Thanks for your open-source code of your project. I have few questions when I use my private data to train the model. My data only have person and background, the foreground of the ground-truth is 255, the background of the ground-truth is 0, and I modify `number_of_classes=1`, but when I use resnet_v2_50 to train or your 16645 model to finetune, I get `Average train loss: 0.0        Global Validation Avg Loss: 0.0      MIoU: 1.0 ` when I begin.

How can I fix that?
Looking forward to your reply!

**Comment by sthalles** [comment]
Hello,

if you look at the first lines of train.py you will see that class ids 255 have special meaning. The code is based on Pascal VOC classes. The easiest thing would be changing your class ids from 255 to 1 for example.

**Comment by JoyLuo** [comment]
Is the numer_of_class should 1 when just 2 class(background and foreground) in dataset?
